packages/app/cypress/e2e/reporter_header.cy.ts

```diff
@@ -24,6 +24,8 @@ describe('Reporter Header', () => {
       cy.get('[data-cy="spec-file-item"]').should('have.length', 3)
       .should('contain', 'dom-content.spec')
 
+      cy.get('body').type('f')
```

Comment:
I'm not sure why the test won't pass as it was, at least not in my setup. I will need to monitor how CI handles this.